app/redirects/releases.tsx

```diff
@@ -0,0 +1,12 @@
+import { LoaderFunctionArgs } from '@remix-run/node';
+import { redirect } from '@remix-run/react';
+
+const CHANNELS = ['stable', 'pre', 'nightly'];
```

Comment:
```suggestion
const CHANNELS = ['stable', 'prerelease', 'nightly'];
```

[The old route code](https://github.com/electron/release-status/blob/cde2cad9831e5da14619b444017b176dfafd75ff/src/routes/releases.js#L65-L67) used `prerelease` rather than `pre`.

Reply:
Hm, new route uses `pre` 😆 Will make a mapping